CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+- fix(flask): make template patching idempotent
```

Comment:
```suggestion
## Unreleased

- fix(flask): make template patching idempotent
```

Reply:
IMO it's fine like this and then when we're ready to release we just add the title. That way when we have contributions or whatever we don't need to ask them to add a title. Also should make reverting commits easier.

Reply:
Not ideal either way though :confused: hopefully reno days come soon

Reply:
`reno` uses `## Unreleased` :p

Reply:
it's not autogenerated? 😓

Reply:
yeah :)

Reply:
to clarify, when you use `reno` to generate a changelog, anything not associated with a release tag gets categorized under "Unreleased"

btw, this suggestion doesn't matter much, you can ignore.

Reply:
Gotcha. Yeah I'm okay with not having to include the title manually for now for the mergeability argument above 🙂